handler/src/main/java/io/netty5/handler/ssl/SslContextBuilder.java

```diff
@@ -584,6 +590,52 @@ public SslContextBuilder startTls(boolean startTls) {
         return this;
     }
 
+    /**
+     * Enables DNS name identification after TLS handshake.
+     * This method has similar effect as if the following snippet of code has been written:
+     * <pre>
+     * {@code
+     * SSLParameters sslParameters = new SSLParameters();
+     * sslParameters.setEndpointIdentificationAlgorithm("HTTPS");
+     * engine.setSslParameters(sslParameters);
+     * }
+     * </pre>
+     *
+     * Note: This parameter makes sense only when new ssl engine created with specified peer host
+     *
+     * @see SslContextBuilder#endpointIdentificationAlgorithm(String)
+     * @see SslContext#newEngine(BufferAllocator, String, int)
+     * @see SslContext#newHandler(BufferAllocator, String, int)
+     */
+    public SslContextBuilder hostNameIdentification(boolean enableHostNameIdentification) {
```

Comment:
There is not much value in a separate builder method that takes a boolean. IIRC, "HTTPS" is enabled by default. Users who want to disable it, can pass an empty string to `endpointIdentificationAlgorithm` to disable it. Consider documenting this ability. Example:
1. User config: https://github.com/apple/servicetalk/blob/bb0ed159d52aed38fff41bae5fd19d3b04f9473e/servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/SslAndNonSslConnectionsTest.java#L187
2. How we set it: https://github.com/apple/servicetalk/blob/50079f71d007d3c43b5152086f3844a3f56eac57/servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/SslUtils.java#L66-L68

Reply:
yes, makes sense, I just wanted to add shortcut because I suspect in most cases "HTTPS" method validation will be used (havent seen LDAPS usages so far)